backend/nokia-innovative-project/src/main/java/com/nokia/library/nokiainnovativeproject/NokiaInnovativeProjectApplication.java

```diff
@@ -30,4 +34,20 @@ public CorsFilter corsFilter() {
 		source.registerCorsConfiguration("/**", config);
 		return new CorsFilter(source);
 	}
+
+    @Profile("production")
+	@Bean
+	public BasicDataSource dataSource() throws URISyntaxException {
+        URI dbUri = new URI(System.getenv("DATABASE_URL"));
```

Comment:
you can get that var from yml or env with ```public... dataSource(@Value("${db.url}") String dbUrl) throws...``` 
https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html